### ISSUE TITLE
Unable to run SCDF task on Kubernetes

### ISSUE THREAD

**Issue by Randhir123** [spring-cloud/spring-cloud-dataflow-server-kubernetes#328]
I'm unable to run simple task on Kubernetes.
SCDF version: 1.6.1.RELEASE
Kubernetes version: 1.11.2

dataflow:>app register --type task --name timestamp --uri docker:springcloudtask/timestamp-task:1.3.1.RELEASE 
Successfully registered application 'task:timestamp'
dataflow:>task create task1 --definition "timestamp"
Created new task 'task1'
dataflow:>task launch --name task1
Command failed org.springframework.cloud.dataflow.rest.client.DataFlowClientException: Failed to resolve application resource: Docker Resource [docker:springcloudtask/timestamp-task:1.3.1.RELEASE]

**Comment by sabbyanandan** [comment]
We have an issue in v1.6.1 wrt Docker artifacts. We have now relaxed the rule and it has been resolved via spring-cloud/spring-cloud-dataflow#2377 - we will release v1.6.2 soon. In the meantime, as a workaround, please register the App with a dummy `--metadata-uri`. 

Example:
> app register --name xfmr --type processor --uri docker:springcloudtask/timestamp-task:1.3.1.RELEASE --metadata-uri maven://org.springframework.cloud.stream.app:cassandra-sink-rabbit:jar:metadata:1.3.1.RELEASE --force

**Comment by chrisjs** [comment]
this is resolved in image `1.6.2.RELEASE`, tested with:

```
app register --type task --name timestamp --uri docker:springcloudtask/timestamp-task:1.3.0.RELEASE
task create task1 --definition "timestamp"
task launch --name task1
```
